src/Simple.OData.Client.Core/ODataClientSettings.cs

```diff
@@ -231,6 +240,12 @@ internal ODataClientSettings(ISession session)
             this.AfterResponse = session.Settings.AfterResponse;
             this.OnTrace = session.Settings.OnTrace;
             this.TraceFilter = session.Settings.TraceFilter;
-        }
+			this.Properties = session.Settings.Properties == null ? null : new Dictionary<string, object>(session.Settings.Properties);
+		}
+
+		public class ExtraProperties
+		{
+			public const string STRINGIZE_DATETIME_VALUES = "StringizeDatetimeValues";
+		}
```

Comment:
These lines don't look correctly indented

Reply:
The project uses spaces for tabs which is not the default at least in my VisualStudio. Added a .editorconfig file for the solution to keep the indentation style used.